modules/core/src/v2/coins/tavaxc.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * @prettier
+ */
+import { BaseCoin, KeyPair, SignedTransaction } from '../baseCoin';
+import { BitGo } from '../../bitgo';
+import { BaseCoin as StaticsBaseCoin } from '@bitgo/statics';
+import { AbstractEthLikeCoin, EthSignTransactionOptions, SignedEthLikeTransaction } from './abstractEthLikeCoin';
+import { AvaxC as AvaxCAccountLib } from '@bitgo/account-lib';
+import { NodeCallback } from '../types';
+
+export class TavaxC extends AbstractEthLikeCoin {
```

Comment:
why don't we extend from `AvaxC` instead? 
The methods `generateKeyPair`, `isValidPub`, `isValidAddress` and `signTransaction` can be inherited

Reply:
I thought about this when doing these changes and I still think is the best approach too. 
The only reason I did not do that is because it's not done for the other coins and I wanted to keep it consistent with that.

I'll do it and update this PR